core/modules/presentation/lib/environment/env.dart

```diff
@@ -7,5 +7,5 @@ part 'env.g.dart';
 @Envied(path: 'lib/environment/.env', obfuscate: true)
 abstract class Env {
   @EnviedField(varName: 'API_URL')
-  static final apiUrl = _Env.apiUrl;
+  static final String apiUrl = _Env.apiUrl;
```

Comment:
do we need that String here?

Reply:
because when I call `melos run rebuild` on a new project I get:
`[presentation]: Obfuscated envied can only handle types such as `int`, `bool` and `String`. Type `InvalidType` is not one of them.
[presentation]: package:presentation/environment/env.dart:10:16
[presentation]:    ╷
[presentation]: 10 │   static final apiUrl = _Env.apiUrl;
[presentation]:    │                ^^^^^^
[presentation]:    ╵
[presentation]: [INFO] Running build completed, took 14.6s
[presentation]: 
`

And because We are discussing about apiUrl, type String isn't a bad choice I think